### PR TITLE
Add whitelist messaging to backup failure and credentials verification screen

### PR DESCRIPTION
**Pull request by manzoorwanijk**
Addresses 1164141197617539-as-1201487671935490

#### Changes proposed in this Pull Request

* Add appropriate messaging on the credentials verification screen to better guide the users.
* Add the similar messaging to Backup section to guide users about their host blocking Jetpack IPs.

#### Testing instructions

* Create a JN site and set up Jetpack Backup
* Goto Settings section on Jetpack Cloud - `jetpack.cloud.localhost:3001/settings/:site`
* Enter your SSH/SFTP credentials but enter a wrong port number (e.g. 100) to simulate a blocked request.
* Click on "Test and save credentials"
* Confirm that the credentials test fails with IP whitelist message under "More details"
* Now may be spin a new JN site and set up Jetpack Free
* Install WordFence plugin to the site and add [Jetpack IPs](https://jetpack.com/support/how-to-add-jetpack-ips-allowlist/) to block list
* Trigger a backup using Rewind Debugger and it should fail because of WordFence blocking.
* Goto Backup section on Jetpack Cloud - `jetpack.cloud.localhost:3001/backup/:site`
* Confirm that you see the backup failed messaging with a link to guide for whitelisting IPs

Screenshot 2022-02-23 at 7 18 07 PM

Screenshot 2022-02-23 at 6 31 00 PM

### CODE REVIEW

File: client/components/jetpack/daily-backup-status/status-card/backup-failed.jsx [hunk 1]
```diff
@@ -26,38 +27,65 @@ const BackupFailed = ( { backup } ) => {
...
 	return (
 		<>
 			<div className="status-card__message-head">
 				<img src={ cloudErrorIcon } alt="" role="presentation" />
 				<div className="status-card__message-error">{ translate( 'Backup failed' ) }</div>
 			</div>
-			<div className="status-card__title">{ getDisplayDate( backup.activityTs, false ) }</div>
+			<div className="status-card__title">
+				{ mayBeBlockedByHost
+					? translate( `We'are having trouble backing up your site` )
```
Comment (elliottprogrammer) [inline client/components/jetpack/daily-backup-status/status-card/backup-failed.jsx]: Just noticed a small typo here in `We'are`.  I think it should read `We're` or `We are`.

**Review by cbauerman — APPROVED** [review]
Looks great! Like @elliottprogrammer said, let's get that typo fixed but otherwise LGTM!

Also, I think a couple of the testing instructions could be more clear. For example, the reason we need to use Jetpack Free on the second site is to connect it, then add Jetpack Backup later after we have blocked the IP range. Adding that rationale is always helpful in a PR!

File: client/components/jetpack/daily-backup-status/status-card/backup-failed.jsx (hunk 1 above)
Comment (manzoorwanijk) [reply client/components/jetpack/daily-backup-status/status-card/backup-failed.jsx]: Oh, thank you for catching that.

**Comment by manzoorwanijk** [comment]
> I think a couple of the testing instructions could be more clear. For example, the reason we need to use Jetpack Free on the second site is to connect it, then add Jetpack Backup later after we have blocked the IP range. Adding that rationale is always helpful in a PR!

Thank you for pointing that out. Will keep in mind.
Just for the sake of clarity, setting up Jetpack Free instead of Jetpack Backup was to avoid a backup being created while setting up WordFence.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7154818

Thank you @manzoorwanijk for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7154818).

**Pull request merged**